crypto/schnorr-adaptor-signature/keys_test.go

```diff
@@ -25,7 +25,8 @@ func FuzzKeyGen(f *testing.F) {
 
 		// ensure that the corresponding btcPK and btcSK
 		// constitute a key pair
-		btcPK := encKey.ToBTCPK()
+		btcPK, err := encKey.ToBTCPK()
+		require.NoError(t, err)
```

Comment:
maybe add a case where the `encKey` is one and assert error?

Reply:
Probably not necessary, as per offline discussion constructor functions of `encKey` enforce Z==1, and this check is only for precaution in case future modifications of the code break this assumption